Update README.md (#17)
wolfi-dev/community: README.md

| Resource    | Details, Links, etc. |
| ----------- | ----------- |

> | Calendar    | Monthly Wolfi Community Calls have been discontinued, but you can review previous calls |

Reviewer: ```suggestion
| Calendar    | Monthly Wolfi Community Calls have been discontinued. |
```

> | Notes | View our [notes](https://docs.google.com/document/d/1wBE3W81Xso6BDOU3-tWzfxGTP_X1HNsdufWbvyycaxE/edit#heading=h.zgngk9ekm1wf) from community meetings |

Reviewer: ```suggestion
| Notes | View the [notes](https://docs.google.com/document/d/1wBE3W81Xso6BDOU3-tWzfxGTP_X1HNsdufWbvyycaxE/edit#heading=h.zgngk9ekm1wf) from past community meetings |
```

| Resource    | Details, Links, etc. |
| ----------- | ----------- |

> | Calendar    | Monthly Wolfi Community Calls have been discontinued, but you can review previous calls |

Author: ```suggestion
| Calendar    | Monthly Wolfi Community Calls have been discontinued. [Review previous calls](https://youtube.com/playlist?list=PLLjvkjPNmuZkqtDoGuV-8SkZw6dwmHxF5) |
```
